jep/1/README.adoc

```diff
@@ -0,0 +1,939 @@
+
+Discussions-To:: [[header-discussions-to]]
+For a JEP where final pronouncement will be made on a list other than
+jenkinsci-dev@, a **Discussions-To** row will indicate the mailing list
+or URL where the pronouncement will occur. A temporary Discussions-To header
+may also be used when a draft JEP is being discussed prior to submission for
+pronouncement.
+
+Requires:: [[header-requires]]
+JEPs may have a **Requires** row, indicating the JEP numbers that this
+JEP depends on.
+
+Superseded-By:: [[header-superseded-by]]
+JEPs may also have a **Superseded-By** row indicating that a JEP has been
+rendered obsolete by a later document; the value is the number of the JEP that
+replaces the current document. The newer JEP must have a **Replaces** row
+containing the number of the JEP that it rendered obsolete.
+
+Resolution:: [[header-resolution]]
+A  **Resolution** section will be added to JEPs when their status is set to
+Accepted, Rejected or Withdrawn.
+It will include a link to the relevant post in the jenkinsci-dev@ mailing list archives.
+
+
+==== Auxiliary Files
+
+JEPs may include auxiliary files such as diagrams. Such files must be
+named appropriately, with lowercase letters and no spaces, and be included in
+the directory with the `README.adoc` describing the JEP.
+
+=== Reporting JEP Bugs, or Submitting JEP Updates
+
+The process for reporting a bug or submiting a JEP update depends on several factors, such
+as the maturity of the JEP, the preferences of the JEP sponsor, and the nature
+of the comments. For the early draft stages of the JEP, it's probably best to
+send  comments and changes directly to the JEP sponsor. For more mature, or
+finished JEPs consider submitting corrections to the JEP repository
+footnoteref:[repo] or the Jenkins issue tracker
+footnoteref:[issues,https://issues.jenkins-ci.org].  If the JEP sponsor is a
+Jenkins developer, assign the bug/patch to them, otherwise assign it to a JEP
+editor.
+
+When in doubt about where to send changes, please check first
+with the JEP sponsor and/or a JEP editor.
+
+
+Even JEP sponsors with git push privileges for the JEP repository should submit
+via Pull Request, with the exception of status or resolution updates
+which may be pushed directly given the change was already discussed
+and agreed to elsewhere.
+
+[[transferring]]
+=== Transferring JEP Sponsorship
+
+It occasionally becomes necessary to transfer sponsorship of JEPs to a
+new sponsor. In general, it is preferable to retain the original sponsor as
+a co-sponsor of the transferred JEP, but that's really up to the
+original sponsor. A good reason to transfer sponsorship is because the
+original sponsor no longer has the time or interest in updating it or
+following through with the JEP process, or has fallen off the face of
+the 'net (i.e. is unreachable or not responding to email). A bad
+reason to transfer sponsorship is because the sponsor doesn't agree with the
```

Comment:
> doesn’t agree with the direction of the JEP

It's unclear to me how this could happen when the sponsor is the author.

Reply:
The sponsor doesn't always write the JEP.
One possible case: I come up with an idea, volunteer as sponsor, start writing it, and then three other contributors come along and poke tons of holes in my implementation or some parts of my idea.  I, as sponsor, should not quit being the sponsor because I don't like their implementation.